xbmc/games/addons/GameClientProperties.h

```diff
@@ -77,7 +77,7 @@ class CGameClientProperties
   bool HasProxyDll(const std::string& strLibPath) const;
 
   const CGameClient* const  m_parent;
-  game_client_properties    m_properties;
+  AddonProps_Game&          m_properties;
```

Comment:
This is one of the biggest changes where the pointer becomes moved there and the static is now in `CGameClient`, during test here has it worked with the addon properties on the classes.

@garbear are the changes here and on this request OK for you? During test here at home has it work. Due to a renamed structure (to match style of other types) need also the game addon a update